### PR TITLE
Inlude post creation signal in resource create endpoint

### PR DESCRIPTION
**Pull request by pkdash**

### CODE REVIEW

**Comment by pkdash** [comment]
@hyi Small PR for you to review. Thanks.

File: hs_core/views/__init__.py [hunk 1]
```diff
@@ -1024,30 +1024,26 @@ def create_resource(request, *args, **kwargs):
...
             files=resource_files,
             fed_res_file_names=fed_res_file_names,
-            fed_res_path = fed_res_path[0] if len(fed_res_path)==1 else '',
+            fed_res_path = fed_res_path[0] if len(fed_res_path) == 1 else '',
             fed_copy_or_move=fed_copy_or_move,
             content=res_title
     )
-    # except Exception as ex:
-    #     context = {'resource_creation_error': ex.message }
-    #     return render_to_response('pages/create-resource.html', context, context_instance=RequestContext(request))
```
Comment (hyi) [inline hs_core/views/__init__.py]: @pkdash Why do you comment out try...except block here? I think try...except block is needed here.

Comment (pkdash) [inline hs_core/views/__init__.py]: @hyi I have not commented out any code here. I have deleted the existing commented out code.

Comment (hyi) [inline hs_core/views/__init__.py]: @pkdash Sorry, I mixed up the github code differences. +1 from me then.

**Pull request merged**